e2e/provision/playbooks/roles/bootstrap/tasks/main.yml

```
@@ -26,9 +26,17 @@
   when: kind.enabled
```

@vjayaramrh what about using this other approach?
```suggestion
  when: k8s.context != "kind-kind"
```

---

We need the equivalent of --tags=all and --tags=real_k8s, would that be achievable?

---

My point is that the same output can be achievable without the manipulation of tags

---

Thanks, Got the point. Can this be something to consider as an enhancement for future? If you agree, I can open an issue to track this.

---

Why not in this PR? This is an enhancement

---

I got the current PR working and tested with the way its is implemented. I am not sure what kind of issues I will run into and testing I need to execute if I made the change now.

---

I agree with @electrocucaracha on this one

---

FYI I shall capture the comment to evaluate the alternate approach in an issue and shall look into it past R3 release. The issue link is at https://github.com/nephio-project/nephio/issues/777